lib/irb.rb

```diff
@@ -528,7 +528,6 @@ def eval_input
             prompt_i = @context.prompt_i.nil? ? "" : @context.prompt_i
             ind = prompt(prompt_i, ltype, indent, line_no)[/.*\z/].size +
               indent * 2 - p.size
-            ind += 2 if continue
```

Comment:
I couldn't see a case where this extra indentation would be desired, but let me know if I'm missing something.  I also wasn't seeing a quick way to add a test for this.